src/servers.jl

```diff
@@ -1,8 +1,8 @@
 
 Base.open(vis::Visualizer, args...; kw...) = open(vis.core, args...; kw...)
 
-function Base.open(core::CoreVisualizer, port::Integer)
-    @async WebIO.webio_serve(Mux.page("/", req -> core.scope), port)
+function Base.open(core::CoreVisualizer, port::Integer, host::IPv4=ip"127.0.0.1")
+    @async WebIO.webio_serve(Mux.page("/", req -> core.scope), host, port)
     url = "http://127.0.0.1:$port"
```

Comment:
Also `host`?

Reply:
I didn't want to change the function signature, in case you use it elsewhere that I missed, so made it a keyword argument. The end function that gets called in Mux.jl https://github.com/JuliaWeb/Mux.jl/blob/1ac39064c91993d49033058d3338538babef46b1/src/server.jl#L56

Reply:
`url = "http://$host:$port"` works. and just results in a `String`; no method signature changes needed.

Reply:
ahh, sorry I misunderstood you. I don't think we can do that in url, a use case can be to open up the server to all interfaces with `host=ip"0.0.0.0"`.